docs/en/config_rover/basic_setup.md

```diff
@@ -122,12 +163,14 @@ You can now continue the configuration process with [rate tuning](rate_tuning.md
 
 ### Differential Specific
 
-| Parameter                                                                                             | Description | Unit |
-| ----------------------------------------------------------------------------------------------------- | ----------- | ---- |
-| <a id="RD_WHEEL_TRACK"></a>[RD_WHEEL_TRACK](../advanced_config/parameter_reference.md#RD_WHEEL_TRACK) | Wheel track | m    |
+| Parameter                                                                                                | Description                               | Unit |
+| -------------------------------------------------------------------------------------------------------- | ----------------------------------------- | ---- |
+| <a id="RD_WHEEL_TRACK"></a>[RD_WHEEL_TRACK](../advanced_config/parameter_reference.md#RD_WHEEL_TRACK)    | Wheel track                               | $m$  |
```

Comment:
FYI only, the styling looks pretty for the units, but I wouldn't have bothered. Sort of thing that is nice if it makes things easier to read or removes ambiguity.